File: VotingModelLibrary/Models/BallotIssue.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace VotingModelLibrary.Models
{
    public class BallotIssue
    {
        [Key]
        public int BallotIssueId { get; set; }
         [Display(Name = "ElectionId")]
        public int ElectionId { get; set; }
        public Election Election { get; set; }
        
        [Display(Name = "BallotIssue")]
        public string BallotIssueTitle { get; set; }
        [Display(Name = "Description")]
        public string Description { get; set; }

        public List<IssueOption> BallotIssueOptions { get; set; }
    }
}

File: Spa/ClientApp/src/app/models/BallotIssue.ts
import { IssueOption } from './IssueOption';

export class BallotIssue {
  ballotIssueId : number;
  electionId: number;
  election: string;
  ballotIssueTitle : string;
  description: string;
  ballotIssueOptions: IssueOption[];
  answer: string;

  constructor(
    ballotIssueId: number,
    electionId: number,
    election: string,
    ballotIssueTitle: string,
    description: string,
    ballotIssueOptions: IssueOption[],
    answer: string
  ) {
    this.ballotIssueId = ballotIssueId;
    this.electionId = electionId;
    this.election = election;
    this.ballotIssueTitle = ballotIssueTitle;
    this.ballotIssueOptions = ballotIssueOptions;
    this.description = description;
    this.answer = answer;
  }
}
